script/generate-cask.rb

```diff
@@ -4,8 +4,13 @@
 FileUtils.rm_rf directories
 FileUtils.mkdir_p directories
 
-Cask::Cask.each do |c|
+Tap.default_cask_tap.cask_files.each do |p|
+  c = Cask::CaskLoader::FromPathLoader.new(p).load
   json_filename = "#{c.token}.json"
   IO.write("_data/cask/#{json_filename}", "#{JSON.pretty_generate(c.to_h)}\n")
   FileUtils.cp "_api_cask.json.in", "api/cask/#{json_filename}"
+
+  html = IO.read "_cask.html.in"
+  html.gsub!("title: $TITLE", "title: \"#{c.token}\"")
```

Comment:
Had to add quotes here because the `0xED` Cask page was showing `237` anywhere that `page.title` was used.